arch/lib/include/asm/atomic.h

```diff
@@ -13,7 +13,10 @@ typedef struct {
 #define ATOMIC64_INIT(i) { (i) }
 
 #define atomic64_read(v)        (*(volatile long *)&(v)->counter)
-void atomic64_add(long i, atomic64_t *v);
+static inline void atomic64_add(long i, atomic64_t *v)
```

Comment:
I guess I also added this function in the latest HEAD: please rebase/merge the latest master branch.